src/Promenade.Web/Views/Shared/LayoutPage.cshtml

```diff
@@ -33,6 +33,18 @@
     </div>
 }
 
+@if (Model.PageLayout.IsPreview)
+{
+    <div class="row">
+        <div class="col-12 col-md-6 offset-md-3">
+            <div class="alert alert-info">
+                You are viewing a preview of how this page will appear as of
+                @Model.PageLayout.StartDate.
```

Comment:
This message could be a bit confusing of there's no start date set for the layout.

Reply:
You are right, will edit to better handle a null start date. I forgot start dates could be null!